src/Utilities/Compiler/CodeMetrics/ComputationalComplexityMetrics.cs

```diff
@@ -403,43 +403,24 @@ public ComputationalComplexityMetrics Union(ComputationalComplexityMetrics other
 
         /// <summary>The number of unique operands found.</summary>
         public long DistinctOperands         //n2
-        {
-            get
-            {
-                return _distinctReferencedSymbols.Count + _distinctReferencedConstants.Count;
-            }
-        }
+            => _distinctReferencedSymbols.Count + _distinctReferencedConstants.Count;
 
         /// <summary>The total number of operator usages found.</summary>
         public long TotalOperators           //N1
         { get; }
 
         /// <summary>The total number of operand usages found.</summary>
         public long TotalOperands            //N2
-        {
-            get
-            {
-                return _symbolUsageCounts + _constantUsageCounts;
-            }
-        }
+            => _symbolUsageCounts + _constantUsageCounts;
 
-        public long Vocabulary               //n
-        {
-            // n = n1 + n2
-            get { return DistinctOperators + DistinctOperands; }
-        }
+        public long Vocabulary               // n = n1 + n2
+            => DistinctOperators + DistinctOperands;
 
```

Comment:
There were 2 comments, one of them was removed.
Luckily, I feel the one that was removed is really extra and unneeded, but the codefix should preserve trivia correctly.

Reply:
I did the removal manually because I wasn't sure how to merge the 2 comments. I have also created an issue on roslyn because the code-fix breaks trivia.s

Reply:
See https://github.com/dotnet/roslyn/issues/50181